test/e2e/kubernetes/kubernetes_test.go

```diff
@@ -92,6 +92,37 @@ var _ = BeforeSuite(func() {
 })
 
 var _ = Describe("Azure Container Cluster using the Kubernetes Orchestrator", func() {
+	Describe("when docker service is stopped", func() {
```

Comment:
you'll need to move this test down to after the SSH key forwarding is set up in order for SSH to work

Reply:
Done, the test is running (locally) now.